cargo-marker/src/main.rs

```diff
@@ -77,7 +110,16 @@ fn main() -> Result<(), ExitStatus> {
             .take_while(|s| s != CARGO_ARGS_SEPARATOR),
     );
 
+    let config = match Config::get_marker_config() {
+        Ok(v) => Some(v),
+        Err(e) => match e {
+            config::ConfigFetchError::NotFound => None,
+            _ => return Err(e.emit_and_convert()),
+        },
+    };
```

Comment:
This should be moved after the flags and version check or maybe better move this into `choose_lint_crates` method, to only load it, if the user doesn't specify lints in the CLI. 

A reason to keep it in main would be, that later `install_driver` might want to also take the configuration, to allow overriding the driver version. But that's in the future and then we can load it here on demand as well.

Either solution is fine, but it should be after the version flag check

Reply:
You're right, nice catch! Totally overlooked it for some reason.